### ISSUE TITLE
webui port is not working

### ISSUE THREAD

**Issue by xbeta** [mcanevet/puppet-btsync#10]
For some reasons, I used this, but the setting for `webui` on listening `0.0.0.0:8989` doesn't work.
Does anyone have the similar problem?

**Comment by darkpixel** [comment]
Yeah.  It doesn't appear to be anything with puppet-btsync--possibly the btsync package that's installed.  I haven't spent much time trying to fix it though--I just ```tail -f``` the log files to see what's going on.
